templates/ngx-service.mustache

```diff
@@ -17,20 +17,25 @@ interface HttpOptions {
   withCredentials?: boolean,
 }
 
+declare var window;
+
 /**
  * Created with ngx-swagger-client-generator (https://github.com/flowup/ngx-swagger-client-generator)
  */
 @Injectable()
 export class ApiClientService {
 
   readonly options: HttpOptions;
-  private domain = 'http://localhost:8080';
+  private domain: string;
 
   constructor(private http: HttpClient,
               @Optional() @Inject('domain') domain: string,
               @Optional() @Inject('httpOptions') options: HttpOptions) {
-    if (domain) {
-      this.domain = domain;
+
+    this.domain = domain;
```

Comment:
Please consider shorthand conditional property assign and string interpolation here:
```
this.domain = this.domain || `//${window.location.hostname ... `;
```